README.md

```diff
@@ -478,6 +501,9 @@ This property previously set the frequency with which Quill polled the DOM for c
 `bounds`
 : Selector or DOM element used by Quill to constrain position of popups. Defaults to `document.body`.
 
+`children`
```

Comment:
Given this change, is `dangerouslySetInnerHTML={{__html:this.state.editorHtml}}` still correct in the toolbar example on line 213?